vimrc

```diff
@@ -75,7 +75,6 @@ if executable('ag')
 endif
 
 " Color scheme
-colorscheme github
 highlight NonText guibg=#060606
```

Comment:
I didn't realize trailing whitespace was highlighted, likely due to my dark color-scheme.